modmesh/apputil.py

```diff
@@ -86,14 +95,19 @@ def get_appenv(name=None):
 get_appenv(name='master')
 
 
-def run_code(code):
+def run_code(code, redirectStdOutFile=None, redirectStdErrFile=None):
```

Comment:
a possible improvement could be recording the history of the outputs.

Reply:
This is not a right way to redirect buffered I/O in either Python or C++.